packages/block-editor/src/components/panel-color-settings/README.md

```diff
@@ -0,0 +1,42 @@
+# `PanelColorSettings`
+
+Render a panel with color choices shown in circles, allowing a user to pick them.
```

Comment:
I think a really important thing to mention here is that `PanelColorSettings` can render multiple color palettes.